cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -275,6 +275,20 @@ class fixed_point {
     return scaled_integer<Rep>{_value, _scale};
   }
 
+  /**
+   * @brief Method that returns the underlying value of the `fixed_point` number
+   *
+   * @return The underlying value of the `fixed_point` number
+   */
+  CUDA_HOST_DEVICE_CALLABLE rep value() const { return _value; }
+
+  /**
+   * @brief Method that returns the scale of the `fixed_point` number
+   *
+   * @return scale_type The scale of the `fixed_point` number
```

Comment:
```suggestion
   * @return The scale of the `fixed_point` number
```
We have the convention of not including the return type in the doxygen `@return` tag
https://github.com/rapidsai/cudf/blob/branch-0.18/cpp/docs/DOCUMENTATION.md#return

Reply:
Yea, sorry I knew about that. My Doxygen VS-Code extension automatically adds it.